src/Admin/AdherentAdmin.php

```diff
@@ -578,6 +579,48 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                 'label' => 'Dernière connexion',
                 'field_type' => DateRangePickerType::class,
             ])
+            ->add('zones', CallbackFilter::class, [
```

Comment:
Seems that this code is repeated in multiple admin classes.
What about create a zone filter type class and use it?

Reply:
done in c4455466ca64271af9d3c18e4c98d4c9ae97c73f